streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -383,6 +390,7 @@ public synchronized void close() {
         if (!open) {
             return;
         }
+        log.trace("Closing store {}", name);
```

Comment:
Similarly, I'd rather log it in the caller function where we could have more context logged (e.g. which thread, which task, is closing this store). And currently since it is only called in `StateStoreManager` which is already logging this event as `log.debug("task [{}} Closing storage engine {}", taskId, entry.getKey());` I think we do not need here.